### ISSUE TITLE
The difference between SynText and ICDAR13

### ISSUE THREAD

**Issue by lucas946** [MhLiao/TextBoxes#95]
I see the format of SynText is different from ICDAR13, the coordinate of boxes in SynText is about 4 points (e.g. [(x1, x2, x3, x4),(y1,y2,y3,y4)]), but the coordinate of boxes in ICDAR13 is about 2 points (x1, y1, x2, y2). I think the difference will cause the change of network structure (e.g. the kernel nums or loss functions and so on). I  don't understand how to pre-train and train model in two different network structure, can you tell me how to  solve this problem?

**Comment by MhLiao** [comment]
All the formats are converted to (xmin, ymin, xmax, ymax), which is the minimum rectangle that covers the quadrilateral.

**Comment by lucas946** [comment]
Your answer help me a lot, Thanks~
